Makefile

```diff
@@ -1,5 +1,4 @@
 LIBDIR := lib
-DISABLE_RIBBON := true
 INDEX_FORMAT := md
 GHPAGES_COMMIT_TTL = 7
 GHPAGES_BRANCH_TTL = 2
```

Comment:
I didn't cut the use of Lucas' mirror for reference XML.

Reply:
Should we?

Reply:
Maybe.  But it is likely that the mirror is still much faster, so I'd want to test it.